Automattic/wp-calypso
# Subscribers Page: add subscribers feature

Closes https://github.com/Automattic/wp-calypso/issues/78363

## Proposed Changes

This PR adds the Add Subscribers button in the top right corner of the Subscribers Page. Clicking on that button will show a popup to enable the user to introduce subscriber emails.

## Testing Instructions

1. Apply this PR and start the application.
2. go to `http://calypso.localhost:3000/subscribers/{the-domain-of-your-site}`. You should see the Add Subscribers button:
<img width="1094" alt="2023-06-19_18-31-15" src="https://github.com/Automattic/wp-calypso/assets/3832570/7013499a-7651-492e-a36e-6225b536dd25">

3. Click the button. You should see the form to enter new subscriber emails.
<img width="775" alt="2023-06-19_18-32-40" src="https://github.com/Automattic/wp-calypso/assets/3832570/01e65e0d-b6ca-4de3-8a3a-b79c435b97cb">

4. Enter some subscriber emails and click the "Add subscribers" button in the form. You should see a confirmation toaster.
<img width="1176" alt="2023-06-19_18-34-07" src="https://github.com/Automattic/wp-calypso/assets/3832570/b8d41e4b-3ff9-4023-bb8b-c33f6cd4170e">

**Note:** The operation is asynchronous, so we cannot update the view now.
